src/subscription_manager/repolib.py

```diff
@@ -120,7 +121,13 @@ def _enable_plugins(cls, pkg_mgr_name, plugin_dir):
                 )
                 continue
 
-            if len(result) == 0:
+            if os.path.isfile("/etc/os-release"):
+                with open("/etc/os-release") as os_release_file:
+                    os_release_data = {}
+                    distros = ["rhel", "centos", "ol", "mageia", "arch", "fedora", "suse"]
+            if os_release_data["ID"] in distros:
+                continue
+            elif len(result) == 0:
```

Comment:
This is no more needed now. Also note that `"/etc/os-release` is actually never read here...